pkg/admin/graphql/identity.go

```diff
@@ -39,10 +39,25 @@ var nodeIdentity = entity(
 			},
 			"claims": &graphql.Field{
 				Type: graphql.NewNonNull(JSONObject),
+				Args: map[string]*graphql.ArgumentConfig{
```

Comment:
What is the intended usage of `names`?

Reply:
The API consumer may only care about some claims, e.g. `username`, `email`. This argument allows consumer to filter the claims, and avoid getting unneeded large claims like OAuth provider profile.

Reply:
I see. It seems like a workaround of the scheme-less situation here.